src/components/SignInCallbackHandler/SignInCallbackHandler.tsx

```diff
@@ -9,7 +9,7 @@ import { useEffect } from 'react';
 import { UserManager } from 'oidc-client';
 
 export interface SignInCallbackHandlerProps {
-    userManager: UserManager;
+    userManager: UserManager | null;
```

Comment:
I think you don't need that anymore

Reply:
the first thing it does is literally
```typescript
    useEffect(() => {
        if (userManager !== null) {
```
and it's we still have nullable user in app side